### DISCUSSION TITLE
Z2M LXC - lost connection after update

### DISCUSSION THREAD

**Discussion by wildchai** [tteck/Proxmox#2194] [Help Needed] [answered]
Hi,

I ran this code in the Z2M LXC Console from the guides to update my Z2M

>  _bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/zigbee2mqtt.sh)"_

I rebooted the Z2M LXC but now I am unable to connect to it via it's IP address. I ran an update a few weeks back with no issues.
Tried to update it again. Still unable to connect.

EDIT: I can ping the z2m server in windows though.

Please advise.

Thanks.

**Comment by tteck** [comment]
What's the output from `systemctl status zigbee2mqtt`?

**Reply by wildchai** [reply, marked as answer]
Thanks for the response. I did some investigating. It was an issue with my outdated external converter. Had to update one of the codes to reflect changes in the new Z2M update.

Appreciate you reading my topic and helping.

Keep up the good work!
